libraries/pom.xml

```diff
@@ -39,12 +39,31 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven-surefire-plugin.version}</version>
                 <configuration>
+                    <testFailureIgnore>true</testFailureIgnore>
+                    <includes>
+                        <include>**/*Test.java</include>
```

Comment:
In Serenity examples, I intentionally wrote a must-fail test to demonstrate how it can be reflected in the Serenity report. Meanwhile, we don't want the automated tests to stop because of this 'must-fail' test. That's why `testFaillureIgnore: true` was added.

I'll mark the test as `@ignored` as I have explained how to reflect the failures in the article.